packages/field/test/FormRegistrarMixin.test.js

```diff
@@ -0,0 +1,71 @@
+import { fixture, html, expect } from '@open-wc/testing';
+import { cache, LitElement } from '@lion/core';
+import { LionFieldset } from '../../fieldset/src/LionFieldset.js';
+import { FormRegistrarMixin } from '../src/FormRegistrarMixin.js';
+
+describe.only('FormRegistrarMixin', () => {
```

Comment:
`only` is very handy to run your unit tests, but should not end up in the pull request :)